Backend/app.py

```
@@ -26,163 +27,41 @@
 
 @app.teardown_appcontext
 def teardown_db(exception):
```

Ruff (0.14.10)

Unused function argument: `exception`

(ARG001)